coralproject/talk
# [CORL-1255] Move wordlist check after staff approval check

## What does this PR do?

Move wordlist check after staff approval check.

This allows staff members to comment anything they want, even if it's on the banned word list.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- login as staff user
- go to admin > config, make sure you have some banned words set
- use the banned words in a comment
- comment should go through and be posted
